3-enrich/stream-enrich/kinesis/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/KinesisEnrich.scala

```diff
@@ -245,14 +244,14 @@ object KinesisEnrich extends Enrich {
    */
   private def lookupDynamoDBEnrichments(
     provider: AWSCredentialsProvider,
-    region: String,
+    endpoint: String,
     table: String,
     keyNamePrefix: String
   ): List[String] = {
     val dynamoDBClient = AmazonDynamoDBClientBuilder
       .standard()
       .withCredentials(provider)
-      .withEndpointConfiguration(new EndpointConfiguration(getDynamodbEndpoint(region), region))
+      .withEndpointConfiguration(new EndpointConfiguration(endpoint, endpoint))
```

Comment:
This is also suspicious

Reply:
Right, they are both set to endpoint since in the beginning we replaced the `region` variable with `endpoint`. As a result, to get the program to compile, I had to go and change every single mention of `region` with endpoint. As per my above comments, I do not think that we need to replace the `region` variable as I reverted this change and everything still worked fine.